### ISSUE TITLE
Building in bash on Windows 10 fails with closure-compile error

### ISSUE THREAD

**Issue by BigBlueHat** [mozilla/webextension-polyfill#25]
```
$ grunt
Running "eslint:src" (eslint) task

Running "eslint:test" (eslint) task

Running "replace:dist" (replace) task
>> 5 replacements in 1 file.

Running "closure-compiler:dist" (closure-compiler) task
Warning: Compilation error Use --force to continue.

Aborted due to warnings.
```

Works fine in the Linux Subsystem (lxss) though. Worth a look for Windows-based builders (like myself).

Thanks for building this!

**Comment by BigBlueHat** [comment]
It looks like this only effects the creation of the `.min.js` file, so even catching/reporting that would be handy/helpful--as you can still use the unminifed one.

**Comment by rpl** [comment]
@BigBlueHat Thanks for reporting this issue, we have recently changed the tool used to generate the minified file (#11) and now we are using babel and the babili-preset instead of uglify (mainly to fix issues like the one described in #10).
Do you mind to try again on a recent master to check if this has been fixed as a side-effect of the  change described above?

**Comment by BigBlueHat** [comment]
Updated and ran `npm run prepublish` and it built fine and all tests passed. Worked like a charm! Thanks, @rpl! ❤️🎩
